en/micro-integrator/docs/references/connectors/google-spreadsheet-connector/google-spreadsheet-connector-example.md

```diff
@@ -0,0 +1,387 @@
+# Google Spreadsheet Connector Example
+
+The Google Sheets API lets users to read and modify any aspect of a spreadsheet. The WSO2 Google Spreadsheet Connector allows you to access the Google Spreadsheet [API Version v4](https://developers.google.com/sheets/api/guides/concepts) through WSO2 EI. It allows users to read/write any aspect of the spreadsheet via the spreadsheets collection.
+
+It has been provided the ability to do Spreadsheet operations and Spreadsheet Data operations. 
+
+## What you'll build
+
+This example explains how to use Google Spreadsheet Connector to create a Google spreadsheet, write data to it and get them read. Further, it explains how the data in the spreadsheet can be edited. 
+
+It will have three HTTP API resources, which are `insert`, `read` and `edit`. 
+
+* `/insert `: The user sends the request payload which includes the name of the spreadsheet should be, what are the sheet names and what data should be inserted to which sheet and the which range of cells.  This request is sent to WSO2 EI by invoking the Spreadsheet API. It creates a spreadsheet with specified data in the specified cell rage. 
```

Comment:
"to which sheet and the which range of cells" some grammar issue.

Reply:
Changed